operator/src/main/resources/instances/developer.properties

```diff
@@ -26,5 +26,6 @@ storage.min-margin=1Gi
 # this is arbitrary, we don't expect request for multiple developer units
 kafka.partition-capacity=1500
 kafka.message-max-bytes=1048588
+kafka.topic-config-enforced-rule=flush.ms:9223372036854775807,index.interval.bytes:4096,compression.type:producer,flush.messages:9223372036854775807,min.cleanable.dirty.ratio:0.5,file.delete.delay.ms:60000,segment.index.bytes:10485760,preallocate:false,unclean.leader.election.enable:false,min.insync.replicas:1
```

Comment:
```suggestion
kafka.topic-config-enforced-rule=flush.ms:9223372036854775807,index.interval.bytes:4096,compression.type:producer,flush.messages:9223372036854775807,min.cleanable.dirty.ratio:0.5,file.delete.delay.ms:60000,segment.index.bytes:10485760,preallocate:false,unclean.leader.election.enable:false,min.insync.replicas:${kafka.scaling-and-replication-factor}
```

Reply:
@k-wall , thanks for the comment. But you might want to check my previous comment: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/778#issuecomment-1199130100 . Simply put, the expression expansion (i.e. `${kafka.scaling-and-replication-factor}`) won't work in [developer|standard].properties. It only works in `application.properties`. In this case, I think we can keep it as is. WDYT?

Reply:
@showuon ok - I appreciated that the property refs wouldn't work across files, but failed to appreciate that it applied with the [developer|standard].properties files too.  Thanks for putting me right.